test/include/test_runner/test_group.h

```diff
@@ -31,6 +36,12 @@ struct TestGroup {
     std::unordered_map<std::string, std::vector<std::unique_ptr<TestStatement>>>
         testCasesStatementBlocks;
     uint64_t bufferPoolSize = common::BufferPoolConstants::DEFAULT_BUFFER_POOL_SIZE_FOR_TESTING;
+    // for multiple connections
+    uint64_t checkpointWaitTimeout =
```

Comment:
This is not really for multiple connections so move this line above the comment. Also @ray6080 should comment if this config will exist in the long term. I feel we might remove it at some point.

Reply:
I think this will exist for quite a long time.